merkato/exchanges/polo_exchange.py

```diff
@@ -0,0 +1,41 @@
+from merkato.exchanges.exchange_base import ExchangeBase
+
+def PoloExchange(ExchangeBase):
```

Comment:
1. is this supposed to be a class?
If so, why replace all util files specific to an exchange with a class for that particular exchange?

Reply:
that is supposed to be `class` not `def`, lol my bad, I'm sleepy
The general idea was to put all functionality specific to an exchange within the interface class for that exchange. It's a self contained component. I'm not sure what's the difference between a util function for an exchange and a non-util, "main" one. Is it just a separation of, private vs public methods? This way, if you know you want to look up something related to interfacing with Tux, you know you want to look in the TuxExchange class.

Reply:
also, this `PoloExchange` class will need to have a buy, sell, etc. method added, just like TuxExchange, currently there aren't any